test/C/src/federated/AfterDelaysWithMultiport.lf

```diff
@@ -0,0 +1,51 @@
+target C {
```

Comment:
A short description of what bug this test catches might help

Reply:
Good suggestion.  Resolved in commit #8100c8420.